code/game/objects/items/storage/uplink_kits.dm

```diff
@@ -116,44 +135,68 @@
 			new /obj/item/clothing/glasses/thermal/syndi(src)
-			new /obj/item/clothing/gloves/color/latex/nitrile(src)
-			new /obj/item/clothing/mask/gas/clown_hat(src)
-			new /obj/item/clothing/under/suit_jacket/really_black(src)
-
-		if("metaops") // 30 tc
-			new /obj/item/clothing/suit/space/hardsuit/syndi(src) // 8 tc
-			new /obj/item/gun/ballistic/automatic/shotgun/bulldog/unrestricted(src) // 8 tc
-			new /obj/item/implanter/explosive(src) // 2 tc
-			new /obj/item/ammo_box/magazine/m12g(src) // 2 tc
-			new /obj/item/ammo_box/magazine/m12g(src) // 2 tc
-			new /obj/item/grenade/plastic/c4 (src) // 1 tc
-			new /obj/item/grenade/plastic/c4 (src) // 1 tc
-			new /obj/item/card/emag(src) // 6 tc
-
-		if("ninja") // 40~ tc worth
-			new /obj/item/katana(src) // Unique , basicly a better esword. 10 tc?
-			new /obj/item/implanter/adrenalin(src) // 8 tc
-			new /obj/item/throwing_star(src) // ~5 tc for all 6
+			new /obj/item/clothing/gloves/fingerless(src)
+			new /obj/item/clothing/head/cowboyhat(src)
+			new /obj/item/clothing/suit/armor/vest/alt(src)
+
+		if("metaops") // ~40 TC. Basically a lone op, theoretically just as rare.
+			new /obj/item/clothing/under/syndicate(src)
+			new /obj/item/radio/headset/syndicate/alt(src)
+			new /obj/item/clothing/glasses/night(src)
+			new /obj/item/storage/belt/military(src)
+			new /obj/item/clothing/suit/space/hardsuit/syndi(src)
+			new /obj/item/tank/jetpack/oxygen/harness(src)
+			new /obj/item/tank/internals/oxygen(src)
+			new /obj/item/gun/ballistic/automatic/shotgun/bulldog/unrestricted(src)
+			new /obj/item/card/emag(src)
+			new /obj/item/card/id/syndicate(src)
+			new /obj/item/ammo_box/magazine/m12g(src)
+			new /obj/item/ammo_box/magazine/m12g(src)
+			new /obj/item/ammo_box/magazine/m12g/slug(src)
+			new /obj/item/ammo_box/magazine/m12g/slug(src)
+			new /obj/item/ammo_box/magazine/m12g/stun(src)
+			new /obj/item/ammo_box/magazine/m12g/stun(src)
+			new /obj/item/implanter/explosive(src)
+			new /obj/item/grenade/plastic/c4 (src)
+			new /obj/item/grenade/plastic/c4 (src)
+			new /obj/item/grenade/plastic/c4 (src)
+
+		if("ninja") // ~30 TC + RTA Budget space ninja
+			new /obj/item/katana(src)
+			new /obj/item/card/emag(src)
+			new /obj/item/implanter/adrenalin(src)
+			new /obj/item/throwing_star(src)
+			new /obj/item/throwing_star(src)
+			new /obj/item/throwing_star(src)
+			new /obj/item/throwing_star(src)
+			new /obj/item/throwing_star(src)
 			new /obj/item/throwing_star(src)
 			new /obj/item/throwing_star(src)
 			new /obj/item/implanter/emp(src)
 			new /obj/item/grenade/smokebomb(src)
 			new /obj/item/grenade/smokebomb(src)
-			new /obj/item/storage/belt/chameleon(src) // Unique but worth at least 2 tc
-			new /obj/item/card/id/syndicate(src) // 2 tc
-			new /obj/item/chameleon(src) // 7 tc
+			new /obj/item/clothing/glasses/phantomthief/syndicate(src)
+			new /obj/item/clothing/suit/armor/reactive/teleport(src)
+			new /obj/item/storage/belt/chameleon(src)
```

Comment:
You could replace the chameleon storage belt with /obj/item/shadowcloak ; this would make it more unique but also somewhat more powerful. Maybe remove the reactive teleport armor if it needs balance.

Reply:
Shadowcloak is literally alpha 1 under the cloak. There is good reason why cloakers were removed.